### PR TITLE
remove unused TODO block for IFS currents coupling

### PR DESCRIPTION
**Pull request by sebastianbeyer**

### CODE REVIEW

File: src/ifs_interface/ifs_interface.F90 [hunk 1]
```diff
@@ -862,24 +818,20 @@ SUBROUTINE nemogcmcoup_lim2_update( mype, npes, icomm, &
    ! Sort out incoming arrays from the IFS and put them on the ocean grid
 
    ! TODO
-   shortwave(:)=0.	! Done, updated below. What to do with shortwave over ice??
```
Comment (sebastianbeyer) [inline src/ifs_interface/ifs_interface.F90:865]: @trackow is this all solved as well?

Comment (trackow) [reply src/ifs_interface/ifs_interface.F90:865]: Yes, there are no open questions really in the coupling. Just make sure that the resetting to 0 is still happening here, the comments can go

**Comment by sebastianbeyer** [comment]
I also changed the misleading abort message here, fits the context.

**Comment by sebastianbeyer** [comment]
Maybe @dsidoren could take a look and then this can be merged?

**Comment by sebastianbeyer** [comment]
Or @koldunovn ?

**Review by koldunovn — APPROVED** [review]

**Pull request merged**